eth/tracers/internal/tracetest/util.go

```diff
@@ -72,6 +72,11 @@ func (c *traceContext) toBlockContext(genesis *core.Genesis, statedb types.State
 		context.Random = &genesis.Mixhash
 	}
 
+	// OP-Stack chains don't support blobs, so return early to avoid panics below
+	if genesis.Config.IsOptimism() {
+		return context
+	}
```

Comment:
nit: it seems a bit safer to put this in the `if` statement below, that way if non-blob conditions are added later we don't need to remember to remove this early return.

Reply:
[2f0528b](https://github.com/ethereum-optimism/op-geth/pull/684/commits/2f0528ba0ed5cde1d7f4b3273caf96f6f70b4a58)